docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,236 @@
+# Percona Monitoring and Management 2.38.0
+
+| **Release date:** | Jun 23, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+### Important security updates
+
+**Oracle Linux 9 as base OS for PMM**
```

Comment:
I've just learned that we use Alma Linux 9 for AMI and OVF. @talhabinrizwan ^

Reply:
Then we need to add this and explain what is used where.

Reply:
@talhabinrizwan Please confirm.

Reply:
this seems, not smart. why wouldn't we base all 3 on the same foundation?  Now we have to be very suspicious of any behavioral changes between docker and AMI/OVF and, particularly with the RHEL source changes, could invalidate our testing.

Reply:
Totally agree, and here is the thread where I've learned that https://perconacorp.slack.com/archives/C057YHXD129/p1687520974217359?thread_ts=1687429976.637339&cid=C057YHXD129

Reply:
@EvgeniyPatlan Please confirm.

Reply:
In this case @BupycHuk @talhabinrizwan, we need to add an appropriate warning in the RN.

Reply:
AFAIR @EvgeniyPatlan told me something about Pricing related to OracleLinux 9 AMIs on AWS. I think thats why we used AlmaLinux 9. He can confirm this by the way.

Reply:
yes there were some technical issues with oracle linux in AWS

Reply:
Per the latest information from @BupycHuk : we getting all images OL9 based 🎉